intel/init.py

```diff
@@ -309,6 +339,51 @@ def assign(cores, pool, count=None):
         c.pool = pool
 
 
+def assign_excl_non_isolcpus(cores, pool, parsed_cpus):
+
+    # Assigns cores provided by the user to an addition pool
+    # called 'exclusive-non-isolcpus'. These cores are
+    # isolated from other user processes but are not placed
+    # in isolcpus
+
+    # Catch all the cores that are not on the system
+    cores_not_on_system = []
+    for c in parsed_cpus:
+        if c >= len(cores):
+            cores_not_on_system.append(c)
+    if len(cores_not_on_system) > 0:
+        raise RuntimeError("Following cores not on system: {}"
```

Comment:
nit: as above - if someone (like myself :wink:) passes logical cores it'll throw an error that cores aren't on the system, while they're listed everywhere, I'd leave like is and inform the user that they need to pass physical core, but otherwise it's alright